### PR TITLE
Create Contributing.md

### PR DESCRIPTION
**Pull request by mihaiep**
Used nodejs/community-committee/master/CONTRIBUTING.md as template

### CODE REVIEW

File: Contributing.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# Node.js Community Contributing Guide 1.0
+
+## Code of Conduct
+
+The Code of Conduct explains the *bare minimum* behavior
+expectations the Node.js Foundation requires from all contributors.
+[Please read it before participating.](https://github.com/nodejs/TSC/blob/master/CODE_OF_CONDUCT.md)
```
Comment (mhdawson) [inline Contributing.md]: Maybe point to the one in this repo ? That way if it gets update you don't need to update this doc as well.

File: Contributing.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+## Vocabulary
+
+* A **Contributor** is any individual creating or commenting on an issue or pull request.
+* A **Collaborator** is a contributor who has been given write access to the repository
+* An **Observer** is any individual who has requested or been requested to attend a CommComm meeting. It is also the first step to becoming a Member.
+* A **Member** is a collaborator with voting rights who has met the requirements of participation to be considered for acceptance, and subsequently voted in by the CommComm voting process.
+
+## Start Here:
+
```
Comment (mhdawson) [inline Contributing.md]: This should probably refer to this repo not the community-committe repo

File: Contributing.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+- Take a look at any issues tagged with [Good First Issue](https://github.com/nodejs/community-committee/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
+
+## Teams and Working Groups
+The Community Committee represents a collection of teams and working groups that are expressly working toward growing and supporting the overall Node.js community. Node.js as a whole needs a broad range of skills and contributions, which are not necessarily technical in nature, to thrive.
```
Comment (mhdawson) [inline Contributing.md]: Don't think we need this section for the version in this repo

File: Contributing.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+by discussion, compromise on the proposed change, or withdrawal of the proposed change.
+
+If a contribution is controversial and collaborators cannot agree about how to get it to land
+or if it should land then it should be escalated to the WG. WG members should regularly
+discuss pending contributions in order to find a resolution. It is expected that only a
+small minority of issues be brought to the WG for resolution and that discussion and
+compromise among collaborators be the default resolution mechanism.
+
+# Becoming a Member and Collaborator
+
```
Comment (mhdawson) [inline Contributing.md]: This should be updated to talk about how to become a member of the user-feedback team

File: Contributing.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+for 3 months, as according to our [governance guidelines](https://github.com/nodejs/community-committee/blob/master/GOVERNANCE.md#section-4-establishment-of-the-community-committee). If you meet the minimum
+attendance and are participating, the CommComm will vote to add you as a member.
+
+All participants added as members should be on-boarded in a timely manner, added as a collaborator, and be given write access to the repository.
+
+Collaborators are expected to follow this policy and continue to send pull requests, go through
+proper review, and have other collaborators merge their pull requests.
+
+# The CommComm Process
+
```
Comment (mhdawson) [inline Contributing.md]: believe this should be case in the context of the user-feedback committee.  Ie The user-feedback committee follows the process of the Community Committee which uses "consensus seeking" process ....

Comment (mihaiep) [reply Contributing.md]: Thank you for the comments/review. I'll take it and work on  patch-2 branch on mihaiep/user-feedback.

**Comment by mikehostetler** [comment]
👍  LGTM

**Pull request merged**